### DISCUSSION TITLE
Windows 10 LTSC 21H2 support

### DISCUSSION THREAD

**Discussion by Forsakenrox** [farag2/Sophia-Script-for-Windows#298] [Ideas]
Need add support a new version of Windows 10 LTSC 21H2

**Comment by farag2** [comment]
You're Nostradamus)
Already. But didn't create a new release. Tomorrow.

<https://github.com/farag2/Sophia-Script-for-Windows/tree/master/Sophia%20Script/Sophia%20Script%20for%20Windows%2010%20LTSC%202021>

Contains a few typos in thhe description. But it's done.

**Reply by Rinentor** [reply]
Good morning.

I would like to take advantage of the thread and ask if there is support for N edition.

Thank you very much,

Greetings.

**Comment by farag2** [comment]
I think there won't any issues with it.

**Reply by farag2** [reply]
The N edition differs from others only be the lack of Media Player.
